portfolio/public/js/index.js

```diff
@@ -0,0 +1,53 @@
+var http = require('http');
+var fs = require('fs');
+var path = require('path');
+// let pathName = __dirname + "/"
+
+var server = http.createServer(requestHandler);
+
+var publicPath = path.join(__dirname, '../..');
+var publicCss =  path.join(__dirname, '../css');
+var  mediaPath = path.join(__dirname, "..")
+console.log(publicPath);
+console.log(publicCss);
+console.log(mediaPath)
+function requestHandler(req, res) {
+    console.log(req.url);
+    
+  // handle all html file together
+  if(req.url === '/') {
+    // set appropriate headers
+    
+    res.setHeader('Content-Type', 'text/html')
+    fs.createReadStream(publicPath+"/templates/index.html").pipe(res);
+    // read file and send chunked data in response
+    // fs.createReadStream(file_path).pipe(res);
+    // for all css files
+  } else if(req.url === '/portfolio'){
+    res.setHeader('Content-Type', 'text/html')
+    fs.createReadStream(publicPath+"/templates/portfolio.html").pipe(res);
+  }
+  else if(req.url.includes('css')) {
+    //handle css file here
+    res.setHeader('Content-Type', 'text/css')
+    fs.createReadStream(publicCss+"/style.css").pipe(res)
```

Comment:
Do not hard code css (style.css) here. 
In case of multiple css it will render style.css for all of different styles linked. 
Instead you can do
```js
// if requested url is '/style.css'
fs.createReadStream(publicCss + req.url ).pipe(res)
```